mix.exs

```diff
@@ -25,7 +25,7 @@ defmodule UeberauthFacebook.Mixfile do
 
   defp deps do
     [{:ueberauth, "~> 0.4"},
-     {:oauth2, "0.6.0"},
+     {:oauth2, "0.8.0"},
```

Comment:
Sure. That makes sense.

My thinking is to leave it locked because of the breaking API changes in oauth2 from 0.6.0 -> 0.7.0. I'm not sure if this will happen again in a future minor version releases and thought it would be better to have it locked than have `mix deps.upgrade` breaking things.